lib/index.js

```diff
@@ -1,5 +1,6 @@
 const Request = require('./request')
-const root = 'https://track.customer.io/api/v1'
+const trackRoot = 'https://track.customer.io/api/v1'
+const apiRoot = 'https://api.customer.io/api/v1'
```

Comment:
Still not quite right, it's a bit confusing because the order of `api/v1/` gets flipped on the `apiRoot` endpoint. So instead of `'https://api.customer.io/api/v1'` it should be `'https://api.customer.io/v1/api'`.

In your defense when I originally added triggered broadcasts to our [API documentation](https://learn.customer.io/api/#apibroadcast_trigger) I also mixed up the order about five or six times before getting it right. 😂

Reply:
hahah good lord, who was responsible for this decision?! oook one more update coming soon

Reply:
ook let's see if this round did it!